code/modules/research/designs/medical_designs.dm

```diff
@@ -482,23 +482,21 @@
 	name = "X-ray Eyes"
 	desc = "These cybernetic eyes will give you X-ray vision. Blinking is futile."
 	id = "ci-xray"
-	build_type = PROTOLATHE | MECHFAB
-	construction_time = 60
+	build_type = MECHFAB
+	construction_time = 120
```

Comment:
for reference you're making it 12 seconds from 6, not 120, times are calculated using deciseconds

Reply:
Oops, oh well.  This is fine, I'll just adjust the description.